code/modules/atmospherics/machinery/components/gas_recipe_machines/atmos_machines_recipes.dm

```diff
@@ -148,3 +148,27 @@ GLOBAL_LIST_INIT(gas_recipe_meta, gas_recipes_list())
 	energy_release = 2900000
 	requirements = list(/datum/gas/antinoblium = 5, /datum/gas/zauker = 20, /datum/gas/bz = 7.5)
 	products = list(/obj/item/stack/sheet/mineral/zaukerite = 2)
+
+/datum/gas_recipe/crystallizer/fuel_pellet
+	id = "fuel_basic"
+	name = "standard fuel pellet"
+	reaction_type = ENDOTHERMIC_REACTION
+	energy_release = 6000000
+	requirements = list(/datum/gas/oxygen = 50, /datum/gas/plasma = 100)
+	products = list(/obj/item/fuel_pellet = 1)
+
+/datum/gas_recipe/crystallizer/fuel_pellet_advanced
+	id = "fuel_advanced"
+	name = "advanced fuel pellet"
+	reaction_type = ENDOTHERMIC_REACTION
+	energy_release = 6000000
+	requirements = list(/datum/gas/hypernoblium = 100, /datum/gas/stimulum = 100)
```

Comment:
Nobody's gonna make this without being on about 20 different types of crack. For a medium-level pellet, it should be medium-level atmos gases. Tritium and Hydrogen, perhaps? Maybe add some BZ so that you dont get all the gases from making a fusion mix.

Reply:
Yeah, i used a wrong recipe from the ones ghil suggested here, will be trit+h2